examples/simple_example/main.tf

```diff
@@ -15,8 +15,10 @@
  */
 
 module "utils" {
-  source  = "terraform-google-modules/utils/google"
-  version = "~> 0.7"
+  source = "../.."
+  #source  = "terraform-google-modules/utils/google"
+  #version = "~> 0.8"
```

Comment:
FYI: This is done automatically by `module-swapper` during the lint and int tests